pkg/csi/service/wcp/controller.go

```diff
@@ -767,8 +774,9 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 		if pvcNamespace, ok := req.Parameters[common.AttributePvcNamespace]; ok {
 			if scName, ok := req.Parameters[common.AttributeStorageClassName]; ok {
 				// Create CNSVolumeInfo CR for the volume ID.
+				capacity := resource.NewQuantity(volSizeMB, resource.BinarySI)
 				err = volumeInfoService.CreateVolumeInfoWithPolicyInfo(ctx, volumeInfo.VolumeID.Id, pvcNamespace,
-					storagePolicyID, scName, vc.Config.Host)
+					storagePolicyID, scName, vc.Config.Host, capacity)
```

Comment:
capacity will be in mb or gb?

Reply:
Capacities like reserved, used are always converted and stored in Mb

Reply:
Changed capacity to bytes format